backend/server/src/models/tournamentModel.ts

```diff
@@ -1,14 +1,22 @@
 import mongoose, { Schema, type Document, type Types } from "mongoose";
-import { type Match } from "./matchModel";
+import type { MatchPlayer, Match } from "./matchModel";
 import { type User } from "./userModel";
-import type { ObjectIdString } from "./requestModel";
 
 export enum TournamentType {
   RoundRobin = "Round Robin",
   Playoff = "Playoff",
   PreliminiaryPlayoff = "Preliminary Playoff"
 }
 
+export interface UnsavedMatch {
+  players: MatchPlayer[];
+  type: string;
+  admin: Types.ObjectId | null;
+  elapsedTime: number;
+  timerStartedTimestamp: Date | null;
+  tournamentRound?: number;
+}
```

Comment:
I would use `Pick` utility type to keep this easily maintainable. This way you don't have to remember to update both of these tables if needed in the future.

Reply:
E.g., pick the necessary properties from the tournament interface